common.gypi

```diff
@@ -102,6 +102,9 @@
         'obj_dir%': '<(PRODUCT_DIR)/obj.target',
         'v8_base': '<(PRODUCT_DIR)/libv8_snapshot.a',
       }],
+      ['target_arch in ("ppc64", "s390x")', {
```

Comment:
```suggestion
      ['target_arch in "ppc64 s390x"', {
```
Your approach is actually more correct but the search-in-string pattern is what's used everywhere else.